README.md

```diff
@@ -9,3 +9,21 @@ protoc --go_out=. --go_opt=paths=source_relative \
   --go-grpc_out=. --go-grpc_opt=paths=source_relative \
```

Comment:
If we remove the precompiled Go files we can move this down to Running (https://github.com/andrewctam/orcanet-market-go/pull/4)

Reply:
Should we also include commands for installing commands necessary for this? Notably, `protoc-gen-go-grpc` is not automatically installed via `go install` even though it's specified in `go.mod`, which causes an error when running this command.

Might be a separate issue that needs fixing, but I haven't found a solution yet.